ui/src/pages/version/components/forms/components/ImageBuilderSection.js

```diff
@@ -0,0 +1,71 @@
+import { FormLabelWithToolTip, useOnChangeHandler } from "@caraml-dev/ui-lib";
+import { 
+    EuiAccordion, 
+    EuiFieldText, 
+    EuiFlexGroup, 
+    EuiFlexItem, 
+    EuiFormRow, 
+    EuiSpacer 
+} from "@elastic/eui";
+
+
+export const ImageBuilderSection = ({
+    imageBuilderResourceConfig,
+    onChangeHandler,
+    errors = {},
+  }) => {
+
+    const { onChange } = useOnChangeHandler(onChangeHandler);
+
+    return (
+    <EuiAccordion 
+    id="imagebuilderOption"
+    buttonContent="Advanced configurations">
+    <EuiSpacer size="s" />
+    <EuiFlexGroup direction="row">
```

Comment:
Nit: I think up until this flex group (the "Advanced configurations" accordion) can just move to `ui/src/pages/version/components/forms/steps/ModelStep.js` and `ui/src/job/form/JobFormOthers.js`, respectively. That should be a part of the parent component while `ImageBuilderSection` only deals with the image builder related components.